cookbooks/fb_syslog/recipes/default.rb

```diff
@@ -77,6 +77,7 @@
   action :start
   subscribes :restart, 'package[rsyslog]'
   # within vagrant, sometimes rsyslog fails to restart the first time
-  retries 5
-  retry_delay 5
+  retries 1
+  retry_delay 15
+  ignore_failure true
```

Comment:
Can we somehow detect the vagrant test environment instead of doing this?

Reply:
the ignore_failure shouldn't be there, that was from testing. I will remove. As to the vagrant stuff I suspect it's not needed anymore and dropped the retries down to only one, but didn't want to remove them altogether in case firstboot_os internally had somehow come to rely on them